ipatests/test_integration/test_commands.py

```diff
@@ -1561,6 +1561,22 @@ def test_ipa_cacert_manage_prune(self):
 
         assert isrgrootx1_nick in result
 
+    def test_ipa_systemd_journal(self):
+        """
+        This testcase checks that administrative user credentials
+        is not leaked to journald log
+        """
+        passwd = "Secret123"
```

Comment:
You should rather use the variables self.config.admin_password and self.config.dirman_password, and make sure that neither one is visible in the journal.

Reply:
@flo-renaud used the variables mentioned.